zzz_generated/app-common/clusters/AccessControl/Ids.h

```diff
@@ -0,0 +1,10 @@
+// DO NOT EDIT MANUALLY - Generated file
+//
+// Identifier constant values for cluster AccessControl (cluster code: 31/0x1F)
```

Comment:
Would be nice to have `0x001F` for the cluster code here if we can, but either way.  Same for other files.  Again, followup is fine.